newswires/client/src/context/SearchContext.tsx

```diff
@@ -247,32 +250,35 @@ export function SearchContextProvider({ children }: PropsWithChildren) {
 		state.queryData?.results,
 	]);
 
-	const handleEnterQuery = (query: Query) => {
-		sendTelemetryEvent(
-			'NEWSWIRES_ENTER_SEARCH',
-			Object.fromEntries(
-				Object.entries(query).map(([key, value]) => [
-					`search-query_${key}`,
-					JSON.stringify(value),
-				]),
-			),
-		);
-		dispatch({
-			type: 'ENTER_QUERY',
-		});
-		if (currentConfig.view === 'item') {
-			pushConfigState({
-				...currentConfig,
-				query,
-			});
-		} else {
-			pushConfigState({
-				...currentConfig,
-				view: 'feed',
-				query,
+	const handleEnterQuery = useCallback(
```

Comment:
this block is just a passby refactor suggested by eslint -- wrapping the function definition in a `useCallback`